src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -37,7 +37,8 @@ public class WasmAppBuilder : Task
 
     // full list of ICU data files we produce can be found here:
     // https://github.com/dotnet/icu/tree/maint/maint-67/icu-filters
-    public string? IcuDataFileName { get; set; }
+    public ITaskItem[]? IcuDataFileNames { get; set; }
```

Comment:
Since the only metadata being read from this is `Identity`, this can be changed to a `string[]`. The values of the array will be the `Identity` values.